src/views/integrations/event-tickets/emails/template-parts/body/event/title.php

```diff
@@ -28,10 +28,12 @@
 <tr>
 	<td class="tec-tickets__email-table-content-event-title-container">
 		<h3 class="tec-tickets__email-table-content-event-title">
-			<?php
-			// phpcs:ignore
-			echo $event->title;
-			?>
+			<a href="<?php echo $event->permalink; ?>" target="_blank">
```

Comment:
Need to escape
```suggestion
			<a href="<?php echo esc_attr( $event->permalink ); ?>" target="_blank">
```

Reply:
Not sure if `esc_url()` would handle `#` correctly. If it does, the `esc_url()` would be better.